frontend-svelte/src/getData.ts

```diff
@@ -0,0 +1,134 @@
+import { initializeApp } from "firebase/app";
+import type { CollectionReference } from "firebase/firestore";
+import {
+    collection, type DocumentData, type Firestore, getDocs, getFirestore
+} from "firebase/firestore/lite";
+
+const firebaseConfig = {
+    apiKey: import.meta.env.COUILLARD_FIREBASE_API_KEY,
```

Comment:
I'm going to approve but paste this back in before merging, because there is no reason to hide this.

Though, it seems like it is not actually necessary for using Firestore, since we do not have `COUILLARD_FIREBASE_API_KEY` in the GitHub environment where the bundle is built, so that value is no doubt undefined in the deployed preview.